ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  IconColor,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import { getURLHost } from '../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  getConnectedSitesList,
+  getOriginOfCurrentTab,
+} from '../../../../selectors';
+import {
+  AvatarFavicon,
+  AvatarFaviconSize,
+  Box,
+  ButtonIcon,
+  ButtonIconSize,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../component-library';
+import { Content, Footer, Header, Page } from '../page';
+import { NoConnectionContent } from './components/no-connection';
+
+export const Connections = () => {
+  const t = useI18nContext();
+  const history = useHistory();
+  const activeTabOrigin = useSelector(getOriginOfCurrentTab);
+  const subjectMetadata: { [key: string]: any } = useSelector(
+    getConnectedSitesList,
+  );
+  const connectedSubjectsMetadata = subjectMetadata[activeTabOrigin];
+  return (
+    <Page data-testid="connections-page" className="connections-page">
+      <Header
+        backgroundColor={BackgroundColor.backgroundDefault}
+        startAccessory={
+          <ButtonIcon
+            ariaLabel={t('back')}
+            iconName={IconName.ArrowLeft}
+            className="connections-header__start-accessory"
+            color={IconColor.iconDefault}
+            onClick={() => history.push(DEFAULT_ROUTE)}
+            size={ButtonIconSize.Sm}
+          />
+        }
+      >
+        <Box
+          display={Display.Flex}
+          alignItems={AlignItems.center}
+          gap={2}
+          justifyContent={JustifyContent.center}
+          className="connections-header__title"
+        >
+          {connectedSubjectsMetadata?.iconUrl ? (
+            <AvatarFavicon
+              name={connectedSubjectsMetadata.name}
+              size={AvatarFaviconSize.Sm}
+              src={connectedSubjectsMetadata.iconUrl}
+            />
+          ) : (
+            <Icon
+              name={IconName.Global}
+              size={IconSize.Sm}
+              color={IconColor.iconDefault}
+            />
+          )}
+          <Text
+            as="span"
+            variant={TextVariant.headingMd}
+            textAlign={TextAlign.Center}
+            ellipsis
+          >
+            {getURLHost(activeTabOrigin)}
```

Comment:
Since we'll be passing the origin as a param in the url in `AppHeader` and `PermissionsPage`, we want to use useParams() to grab the string from the URL like:
```
const urlParams: { origin: string } = useParams();
```
then reference it here:
```
{urlParams.origin}
```